include/mpm_explicit_usf.tcc

```diff
@@ -117,6 +117,16 @@ bool mpm::MPMExplicitUSF<Tdim>::solve() {
         std::bind(&mpm::ParticleBase<Tdim>::compute_strain,
                   std::placeholders::_1, phase, dt_));
 
+    // Assign pressure to nodes
```

Comment:
This whole block should be optional and done only if pressure_smoothing is enabled